dotnetapp-buildcontainer/build.ps1

```diff
@@ -0,0 +1,38 @@
+
+param(
+    [ValidateSet('win', 'linux')]
+    [string]$baseImage = "win"
```

Comment:
>win [](start = 26, length = 3)

We should be consistent across the samples on the terminology used for the nano images.  The other samples are using `nano`.